src/amo/actions/featured.js

```diff
@@ -0,0 +1,27 @@
+import {
+  FEATURED_GET,
+  FEATURED_LOADED,
+  FEATURED_FAILED,
+} from 'core/constants';
+
+
+export function featuredGet({ addonType }) {
```

Comment:
I agree but we haven't been doing that in the past. I'm okay moving toward it; it makes them harder to scan through in a file (here they all stand out as `featured[ACTION]` but I'll move to it here and file an issue to consolidate them.